build.gradle.kts

```diff
@@ -0,0 +1,72 @@
+plugins {
+    kotlin("multiplatform") version "1.6.0"
+}
+
+val GROUP: String by project
+val VERSION_NAME: String by project
+
+group = GROUP
+version = VERSION_NAME
+
+repositories {
+    mavenCentral()
+}
+
+kotlin {
+    jvm {
+        compilations.all {
+            kotlinOptions.jvmTarget = "1.8"
+        }
+        testRuns["test"].executionTask.configure {
+            useJUnitPlatform()
+        }
+    }
+
+    js(BOTH) {
+        nodejs {
+            testTask {
+                useMocha {
+                    timeout = "30000"
+                }
+            }
+        }
+        compilations.all {
+            kotlinOptions {
+                sourceMap = true
+                moduleKind = "umd"
+                metaInfo = true
+            }
+        }
+    }
+
+    val hostOs = System.getProperty("os.name")
+    val isMingwX64 = hostOs.startsWith("Windows")
+    when {
+        hostOs == "Mac OS X" -> macosX64("native")
```

Comment:
This won't work on ARM-based Mac, you also need macosArm64, and probably iOS targets as well (arm64 and x64 for the simulator).

Reply:
Updated to support Mac, iOS and more targets. I do not have a Mac to test any of them, though.

Reply:
Thanks! I can test it